src/Content/Text/BBCode.php

```diff
@@ -2115,6 +2115,32 @@ public static function getTags($string)
 		return array_unique($ret);
 	}
 
+	/**
+	 * Expand tags to URLs
+	 *
+	 * @param string $body 
+	 * @return string body with expanded tags 
+	 */
+	public static function expandTags(string $body)
```

Comment:
Don't we do this somewhere else already?

Reply:
We have a similar functionality here: https://github.com/friendica/friendica/blob/develop/mod/item.php#L392-L432

But that part of the code needs a complete clean up and replacement. This new function will be a part of the clean up job.

Reply:
Fair enough.